lib/recipe/bloc/recipe_bloc.dart

```diff
@@ -891,25 +892,26 @@ class RecipeBloc extends Bloc<RecipeEvent, RecipeState> {
     List<String> newerRecipesRecipeIds = [];
     List<Recipe> existingRecipes = List<Recipe>.from(state.recipes);
 
-    int recipeInfoColumnNumber = 12;
+    int recipeInfoColumnNumber = 13;
     for (List row in data) {
       if (row.length < recipeInfoColumnNumber) {
         emit(state.copyWith(
             recipeImportStatus: 'Import file has too few columns'));
         return;
       }
-      String recipeNumber = row[0];
-      String date = row[1];
-      String recipeName = row[2];
-      String recipeType = row[3];
-      String recipeAttribute = row[4];
-      String recipeMeasurement = row[5];
-      String recipeProbeName = row[6];
-      String recipeProbeAnswers = row[7];
-      String ingredientName = row[8];
-      String ingredientDescription = row[9];
-      String ingredientCookingState = row[10];
-      String ingredientMeasurement = row[11];
+      String employeeNumber = row[0].toString();
```

Comment:
out of curiosity, why does the `toString()` need to be here?

Reply:
This was a bit of a tricky one. If the employee number is purely numeric (am unsure if this is the case for ICRISAT), the csv converter package automatically treats the field as an `int` leading to "int is not String type" errors, hence the explicit conversion. It is safe to call `toString` on a string so this should be fine.

Reply:
understood!

Reply:
although, why don't other fields need this then? don't we have other fields that can be purely numeric, e.g. `ingredientMeasurement` ?

Reply:
ingredientMeasurement consists of `method_value_unit` and hence doesn't run into this issue